### DISCUSSION TITLE
Total race time

### DISCUSSION THREAD

**Discussion by zhvial** [micbay/RaceTimerAndController#3] [Ideas]
It would be cool to record total race time for each racer to see how much time did it take to finish certain amount of laps and show it in racer result screen. For now you can compare only lap times, but wouldn't it be usefull to see how fast you can complete 20 or 30 laps.

**Comment by micbay** [comment]
This is certainly doable. It should be straight forward enough to add an array to hold a running total of how long it has taken each racer to complete their current lapcount.
I'm not sure there is an easy way to make that viewable during a live race without adding a good bit of code to manage automatically cycling through times on the display.
However, it should fit under the racer name on the final results screen for each racer.

**Comment by micbay** [comment]
The current master branch commit (Jan 5, 2023) now includes the total time for each racer on their results screen. This is the total time it took to complete the number of laps finished by that racer.

**Reply by zhvial** [reply]
Thank you very much! Slot car racing fun has tripled with this timer. Once again - great project and project documentation!
